src/ansys/fluent/core/filereader/case_file.py

```diff
@@ -175,9 +198,21 @@ class CaseFile:
         Whether case has particular config var
     """
 
-    def __init__(self, case_filepath: str = None, project_filepath: str = None):
-        """__init__ method of CaseFile class."""
-        if case_filepath and project_filepath:
+    def __init__(
+        self,
+        case_filepath: Optional[str] = None,
+        project_filepath: Optional[str] = None,
+    ) -> None:
+        """Initialize a CaseFile object. Exactly one file path argument must be specified.
+
+        Parameters
+        ----------
+        case_filepath : Optional[str]
+            The path of a case file.
+        project_filepath : Optional[str]
+            The path of a project file from which the case file is selected.
+        """
+        if (not case_filepath) == (not project_filepath):
```

Comment:
This is the only code change in the PR. Added no additional tests and not tested interactively.

Reply:
Curious why this change was made. Isn't `(case_filepath and project_filepath)` significantly clearer than  `(not case_filepath) == (not project_filepath)` when checking whether both were specified? Is this to cover for a potential situation where neither one is specified?

Reply:
_Is this to cover for a potential situation where neither one is specified?_

@raph-luc Yes. Also, I've written the docstring to state _exactly one_.